lib/registries/addon/services/brand.ts

```diff
@@ -0,0 +1,10 @@
+import Service from '@ember/service';
+import Brand from 'ember-osf-web/models/brand';
+
+export default class BrandService extends Service {
+    currentBrand: Brand | null = null;
+
+    setBrand(brand: Brand) {
+        this.set('currentBrand', brand);
```

Comment:
in the octane spirit, would it cause problems to make `currentBrand` a `@tracked` prop, so we don't need to use `set` either? or is that *too* bleeding edge?